nordicsemi/__main__.py

```diff
@@ -1079,7 +1079,11 @@ def get_port_by_snr(snr):
               help='Flash connectivity firmware automatically. Default: disabled.',
               type=click.BOOL,
               is_flag=True)
-def ble(package, conn_ic_id, port, connect_delay, name, address, jlink_snr, flash_connectivity):
+@click.option('-mtu', '--att_mtu',
+              help='ATT MTU. Transmission speed used for ble transfers. Accepted values in range <23, 247>. Default is 247. The error: "Failed to ble_gattc_write. Error code: 3" can in some cases be solved by setting a lower mtu.',
```

Comment:
Suggested rewording of the text:
```suggestion
              help='ATT MTU. Maximum ATT packet size for BLE transfers. '
                   'Accepted values in range [23, 247]. Default is 247. '
                   'Note: Failing DFU transmissions can in some cases be solved by setting a '
                   'lower mtu.',
```